astrapy/db.py

```diff
@@ -1941,6 +1941,7 @@ def create_collection(
         options: Optional[Dict[str, Any]] = None,
         dimension: Optional[int] = None,
         metric: Optional[str] = None,
+        indexing: Optional[Dict[str, Any]] = None,
```

Comment:
shouldn't be better to expose the deny and allow parameters directly ? like

´´´
create_collection("test", allow_indexing=["field1"], deny_indexing=["field2"])
```